packages/react-components/src/components/DevicePermissions/DomainPermissions.tsx

```diff
@@ -59,7 +59,8 @@ export interface CameraAndMicrophoneDomainPermissionsProps extends CommonDomainP
   /** Strings for use with the {@link CameraAndMicrophoneDomainPermissions} */
   strings?: CameraAndMicrophoneDomainPermissionsStrings;
 }
-
+/* @conditional-compile-remove(call-readiness) */
+const isSafari = navigator.userAgent.indexOf('Safari') > -1 && navigator.userAgent.indexOf('Chrome') <= -1;
```

Comment:
what is the behavior of chrome on iphone for this issue? 
Chrome and other browsers on ios are just re-skins of safari so I'm curious if they exhibit the same behavior

Reply:
you may also need to filter out android, firefox and chrome alias: https://stackoverflow.com/a/40463096/3047190

Reply:
Having a hard time to check chrome on iphone, it's shown as unsupported browser and codespace still doesn't work for me :(
Tested the above code on firefox and chrome and safari on Mac, it's behaving properly. The above code already filters out chrome alias

Reply:
That's a neat regex! Updated :) For chrome on iphone still struggling to get codespace working